packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -29,6 +29,7 @@
 
   .main-container {
     flex-grow: 1;
+    padding: var(--terra-consumer-layout-padding, 10px);
```

Comment:
Joey and I discussed this. The issue was if he adds a new css file for a specific example that sets root variables, it affects every example since its all in 1 bundle. 
I thought it would be fine to add a default of 10 and users can always override them. 
Later on we may need to think of a good strategy for this.